### PR TITLE
Feature/deprecated-tag

### PR DESCRIPTION
**Pull request by rubencarvalho**
Added `deprecatedTag` to our deprecated components definitions, to be able to reflect this information in the website.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: test/components/deprecated-tag.test.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import { ComponentDefinition } from '../../src/components/interfaces';
+import { buildProject } from './test-helpers';
+
+let component: ComponentDefinition;
+beforeAll(() => {
+  const result = buildProject('deprecated-tag');
+  expect(result).toHaveLength(1);
+
+  component = result[0];
+  console.log(component);
```
Comment (connorlanigan) [inline test/components/deprecated-tag.test.ts:12]: I'd recommend to remove this `console.log` here.

File: test/components/deprecated-tag.test.ts [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      deprecatedTag: undefined,
+      description: 'Header',
+      inlineType: undefined,
+      name: 'header',
+      optional: true,
+      type: 'string',
+      visualRefreshTag: undefined,
+    },
+  ]);
+  expect(component.regions).toEqual([
```
Comment (connorlanigan) [inline test/components/deprecated-tag.test.ts:38]: Can we test that regions can be deprecated as well?

**Review by connorlanigan — CHANGES_REQUESTED** [review]

File: test/components/deprecated-tag.test.ts (hunk 1 above)
Comment (rubencarvalho) [reply test/components/deprecated-tag.test.ts]: ✅

File: test/components/deprecated-tag.test.ts (hunk 2 above)
Comment (rubencarvalho) [reply test/components/deprecated-tag.test.ts:37]: Updated! 👍

**Review by connorlanigan — APPROVED** [review]

**Pull request merged**